gemfiles/rails_70/Gemfile.lock

```diff
@@ -219,6 +197,7 @@ GEM
 PLATFORMS
   arm64-darwin-22
   java
+  ruby
```

Comment:
Looks like an unintended change, but I can see ruby in arbre:

https://github.com/activeadmin/arbre/blob/181e9ebd4d4a12ae3db7a2f27dc4542d03586ef9/Gemfile.lock#L294

Let me know if this is not a problem

Reply:
I'm not sure what causes it. I'm sure it's fine as is. If it's a problem we can address the removal separately. Thanks!

Reply:
This has been added because I have a setting to prefer ruby builds always on my bundler configuration

```
BUNDLE_FORCE_RUBY_PLATFORM: "true"
```

but should not be a proble